lib/active_shipping/carriers/ups.rb

```diff
@@ -1177,10 +1173,13 @@ def commit(action, request, test = false)
       response.encode('utf-8', 'iso-8859-1')
     end
 
-    def track_commit(action, request, test = false)
+    def track_commit(action, tracking_number, test = false)
       headers = {}
       headers['Authorization'] = "Bearer #{get_cached_bearer_token(test)}"
-      response = ssl_post("#{test ? TEST_URL : LIVE_URL}/#{RESOURCES[action]}", request, headers)
+      headers['transId'] = "678678err789"
```

Comment:
why theses are hardcoded

Reply:
@ankit-tangoe transId we are not using in our flow for tracking, even we are not storing also in our database. But API call requires so i just added one sample data.